source/includes/ImplementationId/implementation_id.md

```diff
@@ -0,0 +1,116 @@
+# Implementation Id Configuration
+
+## Available operations for Implementation Id Configuration
+
+1. [Retrieve Configuration](#retrieve-configuration)
+2. [Update Configuration](#update-configuration)
+
+## Retrieve Configuration
+
+> Retrieve Configuration
+
+```shell
+GET /implementationid
+```
+
+```java
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+Request request = new Request.Builder()
+  .url("/openmrs/ws/rest/v1/implementationid")
+  .method("GET", null)
+  .addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
+  .addHeader("Cookie", "JSESSIONID=1B06650EB0428F51EC119C909F58327C")
+  .build();
+Response response = client.newCall(request).execute();
+```
+
+```javascript
+var requestHeaders = new Headers();
+requestHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
+requestHeaders.append("Cookie", "JSESSIONID=1B06650EB0428F51EC119C909F58327C");
+
+var requestOptions = {
+  method: 'GET',
+  headers: requestHeaders,
+  redirect: 'follow'
+};
+
+fetch("/openmrs/ws/rest/v1/implementationid", requestOptions)
+  .then(response => response.text())
+  .then(result => console.log(result))
+  .catch(error => console.log('error', error));
+```
+
+> Success Response
+
+```response
+{
+    "name": "test",
+    "description": "recovering teest",
+    "implementationId": "test",
+    "passphrase": "test"
+}
+```
+
+Retrieves current configuration.
+
+## Update Configuration
+
+> Update Configuration
+
+```shell
+POST /implementationid
+{
+    "name": "test",
```

Comment:
Can we also change these from `test`?

Reply:
https://github.com/openmrs/openmrs-contrib-rest-api-docs/pull/163/commits/8aa5496224b72e97073f479c83ffa306f1fe7528